execution_engine/src/storage/trie_store/operations/tests/bytesrepr_utils.rs

```diff
@@ -0,0 +1,37 @@
+use casper_types::bytesrepr::{self, FromBytes, ToBytes};
+
+#[derive(PartialEq, Eq, Debug, Clone)]
+pub(crate) struct PanickingFromBytes<T>(T);
```

Comment:
Is this used anywhere else than tests?

Reply:
It's only used in tests, and in fact, it's in tests mod which is `#[cfg(test)]` only